examples/tutorial/tutorial_part_two.ipynb

```diff
@@ -136,18 +158,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "To use your newly deployed model to generate predictions, it needs to be linked to your Clipper application. Let Clipper know that your `\"cifar_demo\"` app should use the `\"birds_vs_planes_classifier\"` model to serve predictions.\n",
+    "To use your newly deployed model to generate predictions, it needs to be linked to your Clipper application. This tells Clipper to route requests for `\"cifar-demo\"` app to the `\"birds-vs-planes-classifier\"` model to make predictions.\n",
```

Comment:
s\...to route requests for clipper demo app...\...to route requests for the clipper demo app...\

Reply:
Done